bitstore/controllers.py

```diff
@@ -33,7 +33,7 @@ def authorize(auth_token, req_payload):
     try:
         # Get request payload
         owner = req_payload.get('metadata', {}).get('owner')
-        dataset_name = req_payload.get('metadata', {}).get('name')
+        dataset_name = req_payload.get('metadata', {}).get('dataset', '')
```

Comment:
Why not just a simple `get` - what is the advantage of defaulting to the empty string.